src/component/page/deltakerliste-veileder/IngenDeltakere.module.scss

```diff
@@ -1,5 +1,12 @@
 .ingenDeltakere {
```

Comment:
Hvor skal denne boksen ligge, sentrert? Jeg tror ikke denne css-en gjør my fornuftig akkurat nå. Kan ikke alt untatt padding og max-width fjernes og resultatet forblir det samme?

Reply:
Den skulle ha vært sentrert ja, og uansett hva jeg gjør blir den ikke det :( Jeg har fått gitt den riktig bredde, men ellers er det ingenting som funker. Fortell gjerne hva jeg må gjøre for å få den sentrert, for jeg har funnet sikkert fem forskjellige måter, og ingen virker :/